src/plugins/opensearch_dashboards_overview/public/components/overview/overview.test.tsx

```diff
@@ -163,46 +164,42 @@ const mockFeatures = [
   },
 ];
 
-const mockBranding = {
-  darkMode: false,
-  mark: {
-    defaultUrl: '/defaultModeUrl',
-    darkModeUrl: '/darkModeUrl',
-  },
-};
+const makeProps = () => ({
```

Comment:
Another one where it's not immediately clear why it's a function call and not a constant object

Reply:
This is a pattern used throughout the unit tests to cheaply guaranty that the props are not tainted by a test.